maintainers/scripts/luarocks-packages.csv

```diff
@@ -1,89 +1,89 @@
-name,server,version,luaversion,maintainers
```

Comment:
Would you consider switching to `json` format or something similar ? This file is edited by humans, some other formats may be easier to edit.

Reply:
For info, I edit the csv with either https://github.com/saulpw/visidata, libreoffice or neovim. I am not fond of json because of the absence of comments (also I like csv brevity) but I am open to changing it to json (in a follow up PR if someone is interested to do it).  I would like the `vim-plugin-names` to support more parameters like the luarocks-packages.csv . Using a similar format for both could be interesting.

Reply:
Now that this is merged, do you want to have a go at this ? https://github.com/vaidik/commentjson/ looks popular enough.
I've also opened an issue https://github.com/NixOS/nixpkgs/issues/136248